### PR TITLE
Upgrade bleve to v1.0.10

### PR DESCRIPTION
**Pull request by lunny**
As title.

### CODE REVIEW

File: models/migrations/v111.go [hunk 1]
```diff
@@ -385,8 +382,15 @@ func addBranchProtectionCanPushAndEnableWhitelist(x *xorm.Engine) error {
 
 	// Find latest review of each user in each pull request, and set official field if appropriate
 	reviews := []*Review{}
-	var page int64
-	for page = 0; page <= totalPages; page++ {
+
```
Comment (techknowlogick) [inline models/migrations/v111.go]: could you remove these migrations changes from this Pr as they are in a different PR

**Comment by 6543** [comment]
@lunny pleace rebase :)

**Comment by zeripath** [comment]
actually the v111.go migration changes were included in a different PR.

**Comment by 6543** [comment]
thats why a rebase would had fixed it too ...

**Review by 6543 — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**